app/controllers/markets_controller.rb

```diff
@@ -0,0 +1,32 @@
+class MarketsController < ApplicationController
+
+  def new    # this is for the get
+    @market = Market.new
+  end
+
+  def update
+    @market = Market.find(params[:id])
+    @market.update(market_create_params[:market])
```

Comment:
It is a good practice to delete extra code before committing. I know as students we like to keep things in to look back on, but in internships (for example) it's expected to not check in commented out code.